parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -193,7 +193,7 @@ extractHyperkube() {
   mv "$path/hyperkube" "/usr/local/bin/kubectl-${KUBERNETES_VERSION}"
 }
 extractKubeBinaries() {
-  KUBE_BINARY_URL=${KUBE_BINARY_URL:-"https://dl.k8s.io/v${KUBERNETES_VERSION}/kubernetes-node-linux-amd64.tar.gz"}
+  KUBE_BINARY_URL=${KUBE_BINARY_URL:-"https://kubernetesartifacts.azureedge.net/kubernetes/v${KUBERNETES_VERSION}/binaries/kubernetes-node-linux-amd64.tar.gz"}
```

Comment:
@mboersma didn't we do this already in a separate PR?

Reply:
We meant to but apparently didn't: #3151 only changed the URL for this in the VHD.

Reply:
> Update KUBE_BINARY_URL default value (missed in #3151)

From PR description